### DISCUSSION TITLE
Missing hashtags in timeline

### DISCUSSION THREAD

**Discussion by Offerel** [pixelfed/pixelfed#5397] [Q&A]
I have added a handful of hashtags to follow them. But on my startpage timeline, I don't see any posts with this hashtags. I see only post from people, i follow. Is there anything I have todo, to see also followed hashtags in my timeline?

**Comment by arjan-s** [comment]
Same problem here with my account on a v0.12.4 instance. Is this really not a problem others are experiencing??

**Comment by rbnvrw** [comment]
I've created an issue, maybe then it gets more attention: https://github.com/pixelfed/pixelfed/issues/5577
